src/python.cc

```diff
@@ -211,6 +212,8 @@ struct BackendState {
   std::unique_ptr<EnvironmentManager> env_manager;
 };
 
+class StubLauncher;
```

Comment:
Can you move the StubLauncher to a separate file named `stub_launcher.cc` and `stub_launcher.h`?